aspnetcore/log-mon/metrics/samples/Program.cs

```diff
@@ -0,0 +1,25 @@
+using OpenTelemetry.Metrics;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddOpenTelemetry()
+    .WithMetrics(builder =>
+    {
+        builder.AddPrometheusExporter();
+
+        builder.AddMeter("Microsoft.AspNetCore.Hosting",
+                         "Microsoft.AspNetCore.Server.Kestrel");
+        builder.AddView("request-duration",
```

Comment:
This counter name has changed to `http-server-request-duration` in preview 5 or preview 6.